src/stores/zkopru.js

```diff
@@ -21,22 +20,26 @@ const DEFAULT_NETWORKS = {
       blockExplorerUrls: ['https://goerli.etherscan.io']
     }
   },
-  '69': {
-    NAME: 'Optimism testnet',
-    WEBSOCKET: 'wss://optimism-kovan.zkopru.network',
+}
+
+if (process.env.NODE_ENV == "development") {
```

Comment:
Why not keep the config for testnet? If we changed to this, we can only support local dev env only.